moped-database/migrations/1730324794396_add_moped_user_saved_views_table/up.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE public.moped_user_saved_views (
+    id serial NOT NULL,
+    description text NOT NULL,
+    url text NOT NULL,
+    query_filters jsonb,
+    created_by_user_id int4 NOT NULL,
+    updated_by_user_id int4 NOT NULL,
+    created_at timestamptz NOT NULL DEFAULT now(),
+    updated_at timestamptz NOT NULL DEFAULT now(),
+    is_deleted boolean NOT NULL DEFAULT false
+);
+
+ALTER TABLE moped_user_saved_views
+ADD CONSTRAINT fk_moped_user_saved_views_created_by FOREIGN KEY (created_by_user_id) REFERENCES moped_users (user_id),
+ADD CONSTRAINT fk_moped_user_saved_views_updated_by FOREIGN KEY (updated_by_user_id) REFERENCES moped_users (user_id);
+
+-- Adding comments for audit fields
+COMMENT ON COLUMN moped_user_saved_views.description IS 'Description entered by the creator of the view';
+COMMENT ON COLUMN moped_user_saved_views.url IS 'URL string associated with the view (may break if database fields or operators are changed)';
+COMMENT ON COLUMN moped_user_saved_views.query_filters IS 'JSON blob of filters that make up the query';
+COMMENT ON COLUMN moped_user_saved_views.created_by_user_id IS 'User ID of the creator of the view';
+COMMENT ON COLUMN moped_user_saved_views.updated_by_user_id IS 'User ID of the last updater of the view';
+COMMENT ON COLUMN moped_user_saved_views.created_at IS 'Timestamp of when the view was created';
+COMMENT ON COLUMN moped_user_saved_views.updated_at IS 'Timestamp of the last update of the view';
+COMMENT ON COLUMN moped_user_saved_views.query_filters IS 'Boolean indicating whether the view has been soft deleted and thereby not rendered in the UI';
```

Comment:
Small typo in the column name here - could you fix? Love these comments by the way! 🙏 ❤️